### DISCUSSION TITLE
s3 compatible storage [Sep 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3740] [Discord Threads]
> 🙋‍♂️ **thvmh**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### There are few storage options available - which one are you looking after?  
#  
> 🙋‍♂️ **thvmh**  
#### Scaleway, was trying to get it as a "serverless" setup on their serverless offering  
#  
> 💁 **o1lab**  
#### Scaleway is available  
#  
> 💁 **o1lab**  
#### Open team and settings  
#  
> 💁 **o1lab**  
#### Then apps  
#  
> 🙋‍♂️ **thvmh**  
#### I was hoping to store the metadata on there as well, by following the production setup. But that seems to not be possible yet?  
#  
> 💁 **o1lab**  
#### ah storing metadata .. you can see if scaleway works with litestream  
#  
> 💁 **o1lab**  
#### since we support litestream to back up the metadata to s3  
#  
> 💁 **o1lab**  
#### so shud be possible as long as litesteram supports it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1011920887018168380)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
